### DISCUSSION TITLE
Set primary folder (Portable version)

### DISCUSSION THREAD

**Discussion by dannbpz** [cyberofficial/Synthalingua#127] [Q&A]
Hi, not sure what am I missing but i get this pop up when clicking on run, and the generate button doesn't do anything from my side

already run set_up_env.exe and have ffmpeg, 7zr and yt-dlp on the main folder

Thanks!

![Screenshot 2024-08-06 234248](https://github.com/user-attachments/assets/0901e094-2a82-4145-957d-f9844ff1fb54)

**Comment by cyberofficial** [comment]
Hello this error means that you didn't select the "transcribe_audio.exe" file. Click the button with the 3 dots " ... " and choose the `transcribe_audio.exe` program.

Setup `set_up_env.exe` will run automatically after you click run.

**Reply by dannbpz** [reply]
Hi, already selected `transcribe_audio.exe` but still have the same error
![Screenshot 2024-08-07 191041](https://github.com/user-attachments/assets/17d3cf90-2f6a-4bec-97a6-116b60a57659)

The CLI version seems fine but takes a lot of time to load even the small model. 
![Screenshot 2024-08-07 191738](https://github.com/user-attachments/assets/cbefcbe5-3e80-4d01-b2cb-63547a3e2038)
Maybe need to add more settings than just --ram 4gb --device cpu ?

I have a Surface Pro 8 (i7-1185G7/16GB) and Python 3.10.9 installed

Thanks for the help!

**Reply by cyberofficial** [reply]
to be clear, you click generate on the ui and the message pops up?

**Reply by dannbpz** [reply]
ah no, the Generate button just doesn't do anything (idk why)

the message pops up when I click Run even after select the "transcribe_audio.exe" file.

**Reply by cyberofficial** [reply]
Could you try this build: https://github.com/cyberofficial/Synthalingua/releases/download/1.0.99995a441/Synthalingua_Wrapper_Aug_7_2024_Build_1.zip

Download this, and replace the files

**Reply by cyberofficial** [reply]
When using the wrapper, 
![image](https://github.com/user-attachments/assets/ea2431b2-f39d-4535-9d8e-6cb1a958a763)

When you do Click generate, click the unhide checkbox and see if it looks something like mine

**Comment by dannbpz** [comment]
Thanks for the build the gui now works! Now the generate button shows the script and run opens the cmd window
![Screenshot 2024-08-07 204834](https://github.com/user-attachments/assets/4999b5de-a25e-445e-a255-4fd00a620164)

I was just wondering if is there a way to generate the Stream En translation text in smaller sentences like subs instead of paragraphs if I can only use base and small models
![Screenshot 2024-08-07 210810](https://github.com/user-attachments/assets/62cab5e2-6b84-472e-a2ef-3eec4967f49a)

should I just ignore those user warnings btw? (FP16/FP32)

**Reply by cyberofficial** [reply]
> should I just ignore those user warnings btw? (FP16/FP32)

Yea, I don't have a custom message for that error, so it just saying it's using a different system in place since your CPU is missing an instruction set for it, not that it's an issue, it's just using a different method that still works. Like a fall back. Not a thing you should worry about.

**Comment by cyberofficial** [comment]
I apologize for missing your other question
> I was just wondering if is there a way to generate the Stream En translation text in smaller sentences like subs instead of paragraphs if I can only use base and small models

You can make smaller texts by choosing a smaller chunk size, https://github.com/cyberofficial/Synthalingua/wiki/Graphical-User-Interface-‐-Information#stream-chunk-size

Chunk size will download "x" amount of chunks from a stream. So you'll need to make lower chunk sizes for shorter inputs.
